README.md

```diff
@@ -1,35 +1,46 @@
 # Morning challenge - Build the Request module
-This week in your project you will be making API calls from the backend. In the past few weeks you have been making API calls from the frontend with the XMLHttpRequest object and it's methods. Now that you are in the backend we would like you to use the Request module which is [available on npm](https://www.npmjs.com/package/request). This workshop is intended to familiarise you with how this module works by asking you to build a simple version of it.
+This week in your project you will be making API calls from the backend. In the past few weeks you have been making API calls from the frontend with the XMLHttpRequest object and methods. Now that you are in the backend we would like you to learn how to make similar request using the Node core [http module](https://nodejs.org/api/http.html). This workshop is intended to familiarise you with how backend requests are made, and practice using callbacks.
 
 ### Time Required: 1 hour
   - 10 mins to read challenge instructions
   - 40 mins to complete the challenge in pairs
   - 10 mins to regroup and talk over the solution
 
 ### Learning outcome
-To become familiar with the Request module and the Node core [http module](https://nodejs.org/api/http.html).
+To become familiar with making API calls from the backend using the Node core [http module](https://nodejs.org/api/http.html).
 
-### What is the Node Request module?
-Request is a very popular module that helps to make HTTP requests as simple as possible and it is built around the Node HTTP core module. [Here](http://stackabuse.com/the-node-js-request-module/) is a brilliant article on the Request module.
+### When would I need to make an API call from the backend?
```

Comment:
Maybe the examples should highlight why students might want to make an API call _from the backend_. E.g bypassing CORS or hiding API keys

Reply:
[Updated](https://github.com/foundersandcoders/mc-request-module-workshop/pull/8/commits/d3c8b3c6b7bc888a43eb793c02c47529fea5d22e)